### PR TITLE
36 character stats

### PR DESCRIPTION
**Pull request by MLHumphries**
Created initial character stats for everyone. Most of our stats are low. I would like everyone to send me what stats you want or the playstyle you would like be fit (strength, magic, all-around).

### CODE REVIEW

File: dunwoody_disaster/CharacterFactory.py [hunk 1]
```diff
@@ -162,81 +168,17 @@ class CharacterFactory:
...
-            "level": 1,
-            "loot": [],
-            "food": [],
-        },
+        }
     }
 
     @staticmethod
-    def createCharacter(name: str, classType: str) -> Character:
+    def createCharacter() -> Character:
```
Comment (chanson02) [inline dunwoody_disaster/CharacterFactory.py:177]: Might aswell just remove this function.

File: main.py [hunk 2]
```diff
@@ -5,14 +5,13 @@
...
 from dunwoody_disaster.CharacterFactory import CharacterFactory, Character
-import dunwoody_disaster as DD
 
 
 class MainWindow(QMainWindow):
     def __init__(self):
         super().__init__()
         self.setWindowTitle("Dunwoody-Disaster")
-        self.setStyleSheet("background-color: #2f2f2f;")
+        self.setStyleSheet("background-color: #2f2f2f; color: #FFFFFF")
```
Comment (chanson02) [inline main.py]: Thank you for this.

File: main.py [hunk 3]
```diff
@@ -58,11 +57,13 @@ def userSelectedCharacter(self, character: Character):
         self.showMapScreen()
 
     def createPlayableCharacters(self) -> list[Character]:
-        cooper = CharacterFactory.createTestChar()
-        cooper.name = "Cooper"
-        cooper.image_path = DD.ASSETS["cooper"]
-
-        return [cooper]
+        # cooper = CharacterFactory.Cooper()
```
Comment (chanson02) [inline main.py:75]:
```suggestion
```

**Review by chanson02 — APPROVED** [review]
Great work! Please make each character have unique stats before merge. @EngineeringDude123 can we get those images soon?

File: dunwoody_disaster/CharacterFactory.py (hunk 1 above)
Comment (MLHumphries) [reply dunwoody_disaster/CharacterFactory.py:177]: I think we need this until we figure out how we are giving the players their weapons. That's currently the only way we have weapons.

Comment (chanson02) [reply dunwoody_disaster/CharacterFactory.py:177]: Oh, right.

**Review by NoahBartell — APPROVED** [review]
lgtm

**Pull request merged**